mozilla-mobile/android-components/components/feature/customtabs/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <string name="mozac_feature_customtabs_exit_button">Return to previous app</string>
     <string name="mozac_feature_customtabs_share_link">Share link</string>
+    <string name="mozac_feature_customtabs_refresh_button">Refresh</string>
```

Comment:
@HarrisonOg Pinging as I see you landed this string.
Could you add a brief comment to give a little more context to translators? It could be interpreted in different ways right now. Thank you!